{{cookiecutter.project_slug}}/k8s/templates/secrets.yaml.template

```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 stringData:
-  AWS_S3_ACCESS_KEY_ID: $AWS_S3_ACCESS_KEY_ID
-  AWS_S3_SECRET_ACCESS_KEY: $AWS_S3_SECRET_ACCESS_KEY
-  AWS_SES_ACCESS_KEY_ID: $AWS_SES_ACCESS_KEY_ID
-  AWS_SES_SECRET_ACCESS_KEY: $AWS_SES_SECRET_ACCESS_KEY
-  POSTGRES_PASSWORD: $POSTGRES_PASSWORD
-  DATABASE_URL: $DATABASE_URL
-  DJANGO_SECRET_KEY: $DJANGO_SECRET_KEY
+  AWS_S3_ACCESS_KEY_ID: op://{{ cookiecutter.project_name }}/$ENVIRONMENT secrets/AWS_S3_ACCESS_KEY_ID
```

Comment:
I'm curious if the space trips things up here.  From my read of how `op inject` works, it would seem to.

Assuming it does, you'd probably need an enclosed secret reference.  Since you're already in Jinja-land with cookiecutter, you'd need to escape everything, something like:

```
  AWS_S3_ACCESS_KEY_ID: {{ "{{" }} op://{{ cookiecutter.project_name }}/$ENVIRONMENT secrets/AWS_S3_ACCESS_KEY_ID {{ "}}" }}
```

Alternatively, moving to an underscore or dash might be a simpler fix.

Reply:
I tested the new Makefile targets on a newly generated project, and surprisingly, `op inject` correctly resolves the op URL with spaces.